packages/storefront/src/pages/component-meta.md

```diff
@@ -75,4 +113,8 @@ export default class Code extends Vue {
     border-radius: 3px;
```

Comment:
suggestion, you can put this style here:

```
    padding: 0.1rem $pds-spacing-static-small;
    display: inline-block;
    cursor: pointer;
    margin: 3px 0;
```
to make it look better:

<img width="1391" alt="Bildschirm­foto 2023-03-10 um 10 08 32" src="https://user-images.githubusercontent.com/2981884/224273581-a10eaf6e-4499-4a34-a1f8-79e673f58922.png">